<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de relations clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Bienvenue sur l'application de gestion de relation.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Que souhaitez vous faire :</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Contact sélectionné :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="contactSelectionneLabel">
          <property name="text">
           <string>Aucun</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="ButtonCreerFiche">
          <property name="text">
           <string>Créer la fiche d'un contact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonModifContact">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Modifier le contact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonRechercherContact">
          <property name="text">
           <string>Chercher un contact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonAjouterEvenement">
          <property name="text">
           <string>Ajouter un événement</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonModifEvent">
          <property name="text">
           <string>Modifier un événement</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonRequete">
          <property name="text">
           <string>Requête sur les contacts</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>546</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionExporter_en_JSON"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuaide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuaide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionExporter_en_JSON">
   <property name="text">
    <string>Exporter en JSON</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
